### ISSUE TITLE
Mise en évidence du message pointé 

### ISSUE THREAD

**Issue by ShiiFu** [zestedesavoir/zds-site#3341]
Sur le forum, les liens vers un message précis d'un sujet (provenant d'une notification par exemple, comme [celui-ci](https://zestedesavoir.com/forums/sujet/4790/liste-des-zeps/?page=1#p96172) sont mis en évidence par la couleur bleu sur la date d'écriture du message : 
![edge](https://cloud.githubusercontent.com/assets/10438692/12629681/369180bc-c549-11e5-9c03-bdb95fd30eea.PNG)
Ici sous Edge (identique sous Chrome)

Mais sous Firefox le message n'est pas mis en évidence : 
![firefox](https://cloud.githubusercontent.com/assets/10438692/12629711/570bb3b2-c549-11e5-9d65-711e3fa96b51.PNG)

**Comment by WinXaito** [comment]
Ceci est la "faute" du navigateur selon moi, à ce que j'ai pu voir, `Edge` et `Chrome` font comme si l'ID indiqué dans l'URL passais en `focus`, ce qui lui applique le style de quand on clique dessus, alors que `Firefox` non.

Dans l'absolu, j'aurais pensé que le comportement de Firefox était le plus juste, mais étant un soucis de navigateur et le fait que ce ne soit pas voulu, je ne sais pas si on peut considéré ceci comme bug.

**Comment by ShiiFu** [comment]
Je suis d'accord. Mais du coup, il est parfois difficile de visualiser le message où l'on devrait reprendre la lecture, notamment quand celui-ci se trouve en bas de page et qu'il n'est plus le premier visible de la page..

**Comment by AmauryCarrade** [comment]
Je suis assez d'accord avec @ShiiFu de l'intérêt de la mise en évidence du message lié, même si elle n'était pas initialement intentionnelle.
Cela dit, s'il est décidé de l'implémenter, je trouverais mieux de le faire [d'une façon similaire à celle-ci](http://lehollandaisvolant.net/?d=2016/01/22/19/50/09-firefox-creer-des-boutons-pour-des-fonctions-supplementaires#idbd4b9c) : encadrer tout le commentaire d'une bordure différente — évidemment, adaptée au style de ZdS, le lien donné n'est qu'un exemple. Un simple bleuissement¹ de la bordure externe du message pourrait faire l'affaire, étant discret mais efficace.

—
¹ Enfin, la couleur principale du site.

**Comment by ShiiFu** [comment]
Je pense en effet que la mise en évidence proposée par @AmauryCarrade serait plus visible et compréhensible que celle qui existe plus ou moins intentionnellement aujourd'hui. 

Question technique : où est ce que ça se situe dans le code ces petits détails front end ?

**Comment by pierre-24** [comment]
Si c'est juste du CSS, [quelque part par ici](https://github.com/zestedesavoir/zds-site/tree/dev/assets/scss). Après, faudrait mettre une petite couche de javascript dessus, je pense.

**Comment by AmauryCarrade** [comment]
Pas forcément, il y a un sélecteur CSS qui pointe l'élément ciblé par l'ancre de l'URL. Ça pourrait suffire.

**Comment by FanJiyong** [comment]
Salut à tous, 

Je suis en accord avec le fait que pour le moment ça n'est pas hyper visuel, et sans ce poste, je n'aurais sans doute jamais remarquer la différence de couleur….

Par contre, j'aimerais en savoir plus sur l'usage. Dans l'exemple ci-dessus il ne s'agit que de message issus des notifications, je suppose que c'est aussi le cas pour le lien "dernier message" (je suis sur firefox au bureau :() est-ce qu'il y a d'autre cas?

**Comment by tleb** [comment]
> Dans l'exemple ci-dessus il ne s'agit que de message issus des notifications, je suppose que c'est aussi le cas pour le lien "dernier message" (je suis sur firefox au bureau :() est-ce qu'il y a d'autre cas?

À chaque fois qu'il y a une ancre sur un message. Donc, MP et contenus aussi j'imagine.
